install_files/securedrop-ossec-server/DEBIAN/postinst

```diff
@@ -29,6 +29,11 @@ case "$1" in
         chown root:${GROUP} ${OSSEC_HOME}/rules/local_rules.xml
         chmod 440 ${OSSEC_HOME}/rules/local_rules.xml
 
+	# Ensure correct gnupg directory permissions and ownership
+	chown -R ossec:${GROUP} ${OSSEC_HOME}/.gnupg
+	find ${OSSEC_HOME}/.gnupg -type f -exec chmod 600 {} \;
+	find ${OSSEC_HOME}/.gnupg -type d -exec chmod 700 {} \;
```

Comment:
This is minor, but it looks like you're using tabs here while the rest of the files uses spaces, which I think is why the indentation is off a bit in the diff above.

Reply:
fixed in c639a0356!